src/components/Fetch/index.js

```diff
@@ -41,6 +41,8 @@ class Fetch extends Component {
       url : searchParams.get('url'),
       wrap: false,
       caseSensitive: false,
+      filterIntersection: false,
+      inverseFilterIntersection: false,
```

Comment:
Let's do away with the inverse filter intersection control/logic, since it's unclear what it does and you can get the same behavior by modifying individual filters